app/src/main/java/org/wikipedia/settings/languages/WikipediaLanguagesFragment.kt

```diff
@@ -298,6 +299,7 @@ class WikipediaLanguagesFragment : Fragment(), MenuProvider, WikipediaLanguagesI
 
     private fun deleteSelectedLanguages() {
         app.languageState.removeAppLanguageCodes(selectedCodes)
+        BreadCrumbCustomLogHelper.logAppLanguageDelete(requireActivity(), selectedCodes)
```

Comment:
Do we really need a special method for this? Won't this be tracked automatically by breadcrumbs? (`longclick` on a list item, then `click` the delete button?)

Reply:
The delete button`menu_delete_selected` just opens the dialog... and the dialog clicks don't get caught..